### PR TITLE
[1LP][RFR] InfraProvider widgetastic flash messages

### PR DESCRIPTION
**Pull request by izapolsk**
changed flash messages to widgetastic ones in infra provider views

{{pytest: cfme/tests/infrastructure/test_providers.py }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 83f56c549f29d201ab21e59990f89dd8e4aba85d:

`cfme/common/provider.py`:
- :bangbang: Line 203:101: **E501** *line too long (104 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/infrastructure/test_providers.py [hunk 1]
```diff
@@ -74,11 +74,7 @@ def test_name_required_validation():
         name=None,
         endpoints=endpoint)
 
-    err = version.pick(
-        {version.LOWEST: "Name can't be blank",
-         '5.6': FlashMessageException})
-
-    with error.expected(err):
+    with error.expected(AssertionError):
```
Comment (mfalesni) [inline cfme/tests/infrastructure/test_providers.py]: In this case, I would recommend to change to pytest.raises instead.

Comment (izapolsk) [inline cfme/tests/infrastructure/test_providers.py]: took into account and fixed.

File: cfme/tests/infrastructure/test_providers.py [hunk 2]
```diff
@@ -58,7 +58,7 @@ def test_add_cancelled_validation():
 def test_type_required_validation():
     """Test to validate type while adding a provider"""
     prov = InfraProvider()
-    with error.expected(FlashMessageException):
```
Comment (psav) [inline cfme/tests/infrastructure/test_providers.py]: Hmmmm, I'm not sure on this, I actually quite liked the older pattern, as it gave you an idea what you expected to fail, and that it was expected.

Comment (izapolsk) [inline cfme/tests/infrastructure/test_providers.py]: @psav, this was @mfalesni request. I can change it back to old patter if you wish.

**Pull request merged**